### PR TITLE
Prohibit clicking already activated tab

### PR DESCRIPTION
**Pull request by LavaCxx**
---

- [x] I understand that contributing to this repository will require me to agree with the [CLA](https://github.com/anyproto/.github/blob/main/docs/CLA.md)

---

### Description
CLS are very annoying things, and would be even more annoying if they could frequently trigger.
It is better to start from prohibiting activated tab from being repeatedly clicked.
before

### What type of PR is this? (check all applicable)

- [ ] 🍕 Feature
- [x] 🐛 Bug Fix
- [ ] 📝 Documentation Update
- [x] 🎨 Style
- [ ] 🧑‍💻 Code Refactor
- [ ] 🔥 Performance Improvements
- [ ] ✅ Test
- [ ] 🤖 Build
- [ ] 🔁 CI

### Added tests?

- [ ] 👍 yes
- [x] 🙅 no, because they aren't needed
- [ ] 🙋 no, because I need help

### Added to documentation?

- [ ] 📜 README.md
- [ ] 📓 [tech-docs](https://github.com/anyproto/tech-docs)
- [x] 🙅 no documentation needed

### CODE REVIEW

**Comment by ra3orblade** [comment]
I think it's better to rework these flags as force flag or initially set tab as null to avoid this onMount logic.

**Comment by LavaCxx** [comment]
> I think it's better to rework these flags as force flag or initially set tab as null to avoid this onMount logic.

Modified.

File: src/ts/component/page/main/store.tsx [hunk 1]
```diff
@@ -340,7 +344,11 @@ const PageMainStore = observer(class PageMainStore extends React.Component<I.Pag
 		};
 	};
 
-	onView (id: View, isInner: boolean) {
+	onView (id: View, isInner: boolean, isChangeTab: boolean = false) {
+		if (!isChangeTab && this.view == id) {
```
Comment (ra3orblade) [inline src/ts/component/page/main/store.tsx:348]: add parenthesis please `&& (this.view == id)`

**Review by ra3orblade — CHANGES_REQUESTED** [review]

**Review by ra3orblade — APPROVED** [review]

**Comment by fuksman** [comment]
@any contributor @LavaCxx code

**Pull request merged**